src/main/kotlin/racingcar/RacingCarApplication.kt

```diff
@@ -0,0 +1,22 @@
+package racingcar
+
+import racingcar.domain.RacingCar
+import racingcar.domain.RacingGame
+import racingcar.domain.RandomGameRule
+import racingcar.view.InputView
+import racingcar.view.ResultView
+
+fun main(args: Array<String>) {
+    val carCount = InputView.readCarCount()
+    val totalRound = InputView.readTotalRound()
+
+    val racingCarGame = RacingGame(
+        gameRule = RandomGameRule,
+        totalRound = totalRound,
+        cars = (1..carCount).map { RacingCar() }
```

Comment:
```suggestion
        cars = List(carCount) { RacingCar() }
```

range 함수가 아닌 List에 있는 함수로도 구현이 가능합니다 :)